Pipfile.lock

```diff
@@ -1,7 +1,7 @@
 {
```

Comment:
Now I understand why it is a mess to read. Because the same file contains both versions and hashes.